docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -0,0 +1,113 @@
+
+This quick start presents a very basic example of auto discovery of nodes, through SNMP.
+Beyond this quick start, you can complete additional tasks to provision your system:
+
+* Use plugins to integrate with external systems.
+* Specify more complex entity detection with OpenNMS detectors (ActiveMQ, DNS, FTP, JDBC, TCP, and so on).
+* Create policies to manage provisioning behavior.
+
+Refer to xref:operation:deep-dive/provisioning/introduction.adoc[Provisioning] in the deep-dive section for details about these and other provisioning-related tasks.
+
+[[provision-snmp-configuration]]
+== Configure SNMP for provisioning
+
+To collect data through SNMP or monitor the availability of SNMP services on a node, you need to configure SNMP for provisioning before using auto or directed discovery.
+This ensures that {page-component-title} can immediately scan newly discovered devices for entities.
+
+Proper SNMP configuration lets {page-component-title} understand network and node topology and automatically enables performance data collection.
+{page-component-title} updates network topology as it provisions nodes.
+
+This quick start procedure illustrates a very basic SNMP configuration using the default parameters.
+You can customize these values for your own network needs.
+
+. In the web UI, click the gear icon in the top right.
+. In the *Provisioning* area, choose *Configure SNMP Community Names by IP Address*.
+. Specify an IP address range in the *First IP Address* and *Last IP Address* fields and accept the default values for the other fields.
+. Click *Save Config*.
++
+This screen sets up SNMP within {page-component-title} for agents listening on IP addresses 10.1.1.1 through 10.254.254.254.
+
+.Configure SNMP community names
+image::provisioning/SNMP_Config.png[SNMP-configuration, 600]
+
+These settings are optimized into the `snmp-config.xml` file.
+
+For an example of the resulting XML configuration, see xref:operation:deep-dive/provisioning/xml-samples.adoc#SNMP-community-xml[Configure SNMP community names].
+
+[[requisition-create]]
+== Create a requisition
+
+A requisition is a set of nodes (networked devices) to import into {page-component-title} for monitoring and management.
+
+Organizing nodes with a similar network monitoring profile into a requisition (for example, routers or switches), lets you assign the same services, detectors, and policies for network monitoring behavior.
+
+A requisition includes a default foreign source definition that serves as a template to define the services to detect on (DNS, FTP, ICMP, and so on), the scan interval for discovery, and the policies to use when provisioning.
+
+While you can perform discovery without a requisition and edit it later, we recommend that you create a requisition before discovery.
+You can iteratively build a requisition and later actually import the requisition's nodes into {page-component-title} to process all add, changes, and deletes at once.
```

Comment:
```suggestion
You can iteratively build a requisition and import the requisition's nodes into {page-component-title} later.
Doing this will process all additions, changes, and deletions at once.
```